### PR TITLE
Add optimism goerli chain

### PR DESCRIPTION
**Pull request by LeifuChen**
`wagmi@0.6.2` add optmism goerli chain.
ref: https://github.com/wagmi-dev/wagmi/releases/tag/wagmi%400.6.2

### CODE REVIEW

**Comment by platocrat** [comment]
Is there an ETA for when this will be merged?

File: packages/rainbowkit/src/components/RainbowKitProvider/provideRainbowKitChains.ts [hunk 1]
```diff
@@ -71,6 +72,7 @@ const chainMetadataByName: Record<ChainName, ChainMetadata | null> = {
   mainnet: { chainId: 1, ...ethereumIcon },
   optimism: { chainId: 10, ...optimismIcon },
   optimismKovan: { chainId: 69, ...optimismIcon },
+  optimismGoerli: { chainId: 420, ...optimismIcon },
```
Comment (nickbytes) [inline packages/rainbowkit/src/components/RainbowKitProvider/provideRainbowKitChains.ts:75]: Chain ID looks right to me: https://community.optimism.io/docs/useful-tools/networks/#optimism-goerli

**Comment by nicoglennon** [comment]
nice @LeifuChen thanks for the PR! only thing is it is currently failing CI due to a small linting error:
> Expected object keys to be in insensitive ascending order. 'optimismGoerli' should be before 'optimismKovan'  sort-keys-fix/sort-keys-fix

if you just move both your additions above the `optimismKovan` lines it should be good. if you want to make the change, i will merge once it passes CI 🤠

**Comment by LeifuChen** [comment]
> nice @LeifuChen thanks for the PR! only thing is it is currently failing CI due to a small linting error:
> 
> > Expected object keys to be in insensitive ascending order. 'optimismGoerli' should be before 'optimismKovan'  sort-keys-fix/sort-keys-fix
> 
> if you just move both your additions above the `optimismKovan` lines it should be good. if you want to make the change, i will merge once it passes CI 🤠

Updated. Thanks.

**Review by nicoglennon — APPROVED** [review]

**Comment by peduarte** [comment]
hey, does this need a [docs change](https://www.rainbowkit.com/docs/chains) update as well?

**Comment by nicoglennon** [comment]
thanks for catching that. will add  now 👍

**Pull request merged**